husky_components/select/main.js

```diff
@@ -1100,14 +1101,18 @@ define([], function() {
 
             // callback, if defined
             if (!!this.options.selectCallback) {
-                this.options.selectCallback.call(this, key);
+                this.options.selectCallback.call(this, key, selectedValue);
             } else {
-                this.sandbox.emit(EVENT_SELECTED_ITEM.call(this), key);
+                this.sandbox.emit(EVENT_SELECTED_ITEM.call(this), key, selectedValue);
             }
         },
 
         // triggers select callback or emits event
         triggerPreSelect: function(key) {
+            if (!this.options.emitPreSelect) {
+                return;
+            }
```

Comment:
Why do we need to configure if this event is emitted? Couldn't we simply not listen to the event? If there is no good reason for that, I would omit it, since it makes stuff unnecessarily complicated.